### ISSUE TITLE
Data set directory & How to read it?

### ISSUE THREAD

**Issue by whuchris** [diegomorin8/Deep-Neural-Networks-for-Piano-Music-Transcription#1]
Sorry to bother, I want to know the detail about the directory when I run the data processing procedure. Exactly, when i  run the CreateList.py and the others in the directory Preprocessing, how to organize the dataset directory and set the args for the program. My file directroy is like the following.
Deep-Neural-Networks-for-Piano-Music-Transcription
--Remote   //I use the remote mechine to run the code
----DNN
----LSTM
----‘Others‘’
----Preprocessing
--Maps
----AkPnBcht
------MUS
--------'dataset'
I use only one kind of the dataset to test the code. I'm a newcomer for AI. Thank you very much. By the way, should i unzip the dataset though i've already done it. ;)

**Comment by SheydaGhb** [comment]
I have the exact question , what are the sys.arg values for data pre-processing. it is not clear 
thanks

**Comment by liam637** [comment]
yes I would also like to know the input arguments if someone could follow up on that

**Comment by signornessina** [comment]
i have the same question, could someone help us?
